frontend/src/views/web/challenge/my-submission.html

```diff
@@ -53,7 +53,7 @@
                         </div>
                     </div>
                 </div>
-                <div class=" ev-card-body exist-team-card">
+                <div class="ev-challenge-view ev-card-panel exist-team-card">
```

Comment:
Don't remove `ev-card-body` from here.
Don't add `ev-challenge-view` here.
Do these changes and then paste a screenshot of result please
@NSDCars5

Reply:
@aka-jain Made some more changes.